apps/nextjs-app/src/features/demo/components/PoemGrid.tsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Any is always a bad idea....

Reply:
This will be deleted anyway. Hopefully tomorrow. I added this here only to test the example build and it was not good for our dockerization process. I just quick fixed them all.